file-formats/ntty/type/zif_aff_ntty_v1.intf.abap

```diff
@@ -0,0 +1,70 @@
+INTERFACE zif_aff_ntty_v1
+  PUBLIC.
+
+
+  TYPES:
+    "! $values { @link zif_aff_ntty_v1.data:co_profile }
+    "! $default { @link zif_aff_ntty_v1.data:co_profile.text }
+    ty_profile TYPE c LENGTH 20,
+
+    "! $minimum 0
+    "! $maximum 2147483647
+    ty_max_length TYPE i,
+
+    "! <p class="shorttext">General Information</p>
+    "! General information
+    BEGIN OF ty_general_information,
+      "! <p class="shorttext">Language Dependent</p>
+      "! Language dependent
+      "! $showAlways
+      language_dependency TYPE abap_bool,
+      "! <p class="shorttext">Multiple Notes</p>
+      "! Multiple notes possible
+      "! $showAlways
+      multiple            TYPE abap_bool,
+      "! <p class="shorttext">Hide Title</p>
+      "! Hides the title
+      no_subject          TYPE abap_bool,
```

Comment:
Here, title/description are very different from the field name. What about 
```suggestion
      hide_title          TYPE abap_bool,
```
?